[ADD] crm: add marketing attribution doc

pabr-odoo opened the pull request:
Add a new document to CRM using Lead Analysis to create marketing attribution reports.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/9566).

pabr-odoo commented:
Hi @StraubCreative just tagging you here to do an initial review of the new doc. Thanks!

StraubCreative reviewed (changes requested):
Hi @pabr-odoo 

Off to a good start! Overall I think you captured the steps/structure well, and next I want to really expand out every section, with a few corrections here and there.

Please see below for comments on how I think we can dial in the use case, so the content is more specific to this kind of report, and contextual to larger sales/marketing ops.

Happy to answer questions as you have them, provide resources like screen recordings, or do a meet/walkthrough, etc., just let me know ^^

In any case, after this next round of revisions, please tag me again for another look, thanks!

StraubCreative commented:
@pabr-odoo also for the merge conflict here on `track_leads.rst` I can show you how to fix this if you want too, lmk!

pabr-odoo commented:
Hey @StraubCreative I finished addressing your last round of suggestions. Please give it another look when you get the chance and let me know if anything needs changing. Thanks!

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+The :doc:`link tracker <../../../websites/website/reporting/link_tracker>` in Odoo can be used to
+create and configure UTMs.
+
+UTMs can also be automatically generated by :doc:`Email Marketing
```
```suggestion
UTMs can also be automatically generated by the :doc:`Email Marketing
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+<../../../marketing/email_marketing>` and :doc:`Marketing Automation
+<../../../marketing/marketing_automation>` apps.
+
+The UTM parameters used in a marketing attribution report are *Medium*, *Source*, and *Campaign* in
```
```suggestion
The UTM parameters used in a marketing attribution report are *Medium*, *Source*, and *Campaign*, in
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+descending order of coverage.
+
+- *Medium* is the UTM with the widest coverage, such as social media.
+- *Source* is more narrow and covers parameters such as the specific social media platform.
```
```suggestion
- *Source* is more narrow, and covers parameters, such as the specific social media platform.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+
+- *Medium* is the UTM with the widest coverage, such as social media.
+- *Source* is more narrow and covers parameters such as the specific social media platform.
+- *Campaign* is the narrowest and can track specific marketing campaigns.
```
```suggestion
- *Campaign* is the most narrow, and can track specific marketing campaigns.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+Create reports
+==============
+
+To start creating a report, click the :icon:`fa-caret-down` :guilabel:`(down caret)` to the right of
```
```suggestion
To start creating a report, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of
```
I know this hasn't been "set in stone" or anything, but I am always "in the camp" of making things as universally-understood as possible. Especially with highly technical jargon, such as 'caret' (arrow) or 'modal' (pop-up)...I just feel like the average reader wouldn't exactly know that term off-the-cuff. In other words, they may actually look for a 'carrot' icon (despite the spelling difference). All that said, I think it's more useful to the reader, and the doc, in general, by using the term "arrow" in lieu of "caret" for those reasonings. What you did was *technically* spot-on, 100% correct -- I'm just suggesting this difference to appease the otherwise not-as-tech-savvy reader.

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+==============
+
+To start creating a report, click the :icon:`fa-caret-down` :guilabel:`(down caret)` to the right of
+the search bar to see the list of filtering and grouping parameters.
```
```suggestion
the :guilabel:`Search...` bar to see the list of filtering and grouping parameters.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+the search bar to see the list of filtering and grouping parameters.
+
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only
+the results that fit the filter. For example, selecting the :guilabel:`Won` filter will only
+show won leads in the attribution report. :guilabel:`Group By`, found in the middle column, is
+used to organize the results into groups and can be used with or without filters.
```
```suggestion
the results that fit the filter. For example, selecting the :guilabel:`Won` filter only
shows leads that have been won in the attribution report.

:guilabel:`Group By`, found in the middle column, is used to organize the results into groups, and can be
used with or without filters.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+   :alt: Select any number of filters and groups in the search options.
+
+.. tip::
+    Setting multiple Group By options will create nested groups according to which is selected
+    first. For example, selecting :guilabel:`Medium` followed by :guilabel:`Source` and then
+    ::guilabel:`Campaign` in :guilabel:`Group By` will sort all results *first* by medium, *then* by
+    the specific sources in each medium, followed by the campaigns in each source.
```
```suggestion
    Setting multiple :guilabel:`Group By` options creates nested groups, according to which option is selected
    first. For example, selecting :guilabel:`Medium`, followed by :guilabel:`Source`, and then
    ::guilabel:`Campaign`, in :guilabel:`Group By` column, sorts all results *first* by medium, *then* by
    the specific sources in each medium, followed by the campaigns in each source.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+    ::guilabel:`Campaign` in :guilabel:`Group By` will sort all results *first* by medium, *then* by
+    the specific sources in each medium, followed by the campaigns in each source.
+
+    This can be verified by looking at the direction of the carat in the group tile in
+    the search bar.
```
```suggestion
    This can be verified by looking at the direction, and order, of the selections in the group tile that appears in
    the :guilabel:`Search...` bar.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+.. example::
+    For a useful first report:
+
+    #. Select the :guilabel:`Active` filter to view only leads that are still marked as active.
+    #. Select :guilabel:`Source` followed by the :guilabel:`City` or :guilabel:`Country`
+       groups depending on which grouping is more relevant.
```
```suggestion
    #. From the :guilabel:`Filters` column, select the :guilabel:`Active` filter to view only leads that are still marked as active.
    #. From the :guilabel:`Group By` column, select (in this specific order) :guilabel:`Source`, followed by the :guilabel:`City` or :guilabel:`Country`, depending on which grouping is more relevant.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+       :align: center
+       :alt: Each lead is now sorted by source, followed by city or country.
+
+    This report will contain all active leads grouped first by the source of the lead, then the
```
```suggestion
    This report contains all active leads, grouped first by the source of the lead, then the
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+       :alt: Each lead is now sorted by source, followed by city or country.
+
+    This report will contain all active leads grouped first by the source of the lead, then the
+    country or city each lead is from. This is useful to see the density of active opportunities
```
```suggestion
    city or country each lead is from. This is useful to see the density of active opportunities
```
suggesting this minor reordering to fall in line with the sentence formatting above

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+    country or city each lead is from. This is useful to see the density of active opportunities
+    sorted by location.
+
+    With this data, marketing campaigns such as conferences or billboards can be targeted to the
```
```suggestion
    With this data, marketing campaigns, such as conferences or billboards, can be targeted to the
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+graph visualization displays the number of leads generated, by month and by sales team, with each
+sales team attributed to its own color per bar (month) shown.
+
+Switch the view to the :icon:`oi-view-list` :guilabel:`(List)` option, by clicking the respective
```
```suggestion
Switch the view to the :icon:`oi-view-list` :guilabel:`(list)` option, by clicking the respective
```
It's my understanding that we are only meant to capitalize Kanban, when it comes to views... but a quick double-check with SALI about this may be best

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+   :align: center
+   :alt: Open the CRM app and click on the Reporting tab at the top of the page, then click Leads.
+
+The :icon:`fa-area-chart` :guilabel:`(Graph)` view will be shown by default, with :guilabel:`Active
```
```suggestion
The :icon:`fa-area-chart` :guilabel:`(graph)` view is shown, by default, with :guilabel:`Active
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+
+To set measures of the report, begin by navigating to the :icon:`oi-view-pivot`
+:guilabel:`(pivot view)`. Click the :guilabel:`Measures` button to view the available measures of
+the report. Select the desired measures, and verify the measures, filters, and groups are all
+displayed correctly in the pivot table.
```
```suggestion
the report. Select the desired measures from the drop-down menu (multiple measures can be selected), and verify the measures, filters, and groups are all displayed correctly in the pivot table.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+the report. Select the desired measures, and verify the measures, filters, and groups are all
+displayed correctly in the pivot table.
+
+Next, click on the :icon:`fa-cog` :guilabel:`(settings)` button to
```
```suggestion
Next, click on the :guilabel:`Actions` :icon:`fa-cog` :guilabel:`(gear)` icon, located to
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+displayed correctly in the pivot table.
+
+Next, click on the :icon:`fa-cog` :guilabel:`(settings)` button to
+the right of :guilabel:`Lead Analysis` in the top-left of the page to view the available export
```
```suggestion
the right of :guilabel:`Lead Analysis` in the top-left of the page, to view the available export
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+the right of :guilabel:`Lead Analysis` in the top-left of the page to view the available export
+options.
+
+To export the report in the Odoo *Documents* app, navigate to :menuselection:`Settings app -->
+Spreadsheet --> Insert list in spreadsheet`. A menu will appear with export options.
```
Does the user *have to* leave the CRM app (and report they just built) to do this? Or, can they simply click the 'Insert in Spreadsheet' button? Also, in the previous paragraph, you instructed the reader to open the Actions (gear) menu...but then didn't break down their options on that menu, nor did you provide information about what they should select... I think it might benefit the overall doc if some added clarification and elaboration was placed in that portion...just to round things out, so to speak...

ksc-odoo replied:
I'm also not seeing a clickable header menu option, feature, or section on the Settings app with the name: Spreadsheet.

ksc-odoo replied:
Perhaps, instead of "Settings app..." you meant to say "Settings icon" (which I have changed to the 'Actions' (gear) above)
In that case, a potential rewording of this section could be...

From the resulting :guilabel:`Actions` :icon:`fa-cog` :guilabel:`(gear) drop-down menu, select :menuselection:`Spreadsheet --> Link menu in spreadsheet'.

Was that what you intended here? If not, just let me know. But, that's what I'm seeing on the runbot I am using for this review.

![Screenshot 2024-07-15 093623](https://github.com/user-attachments/assets/2304e4ec-b037-49ae-8429-b1bea43c86af)

ksc-odoo replied:
UPDATE: I think I see where the "disconnect" happened here...

So, at the top of this section, you instruct the reader to enter Pivot view, and make some customizations. 

Then, you tell them to open the Actions (gear) drop-down menu, but don't tell them why/what to select...

After that, you (I believe mistakenly) instruct them to go to the Settings app, but I don't think that's necessary, as the 'Insert in Spreadsheet' options are available directly from the reporting page.

Following that, you have different results and options via the Spreadsheet directions, and I think that's because the directions you have provided here, are for things in a List view (*not* Pivot view, which you have just told the reader to enter). That's why your instructions/options are different.

When in list view, the "Name of the list..." field and "Insert the first ______ records" field appears on the pop-up...that does not occur when in Pivot...

So, during your revisions, give this section another read-through/run-through, and make the necessary adjustments to ensure that everything lines and works as intended. Thanks! 👍

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+To export the report in the Odoo *Documents* app, navigate to :menuselection:`Settings app -->
+Spreadsheet --> Insert list in spreadsheet`. A menu will appear with export options.
+
+The report can be named using the `Name of the list` field. The number of items on the report can be
```
```suggestion
The report can be named using the `Name of the link` field. The number of items on the report can be
```
If my suggestions aren't lining up with your intentions, just let me know. But this is what I'm seeing during my run-through...

![Screenshot 2024-07-15 094056](https://github.com/user-attachments/assets/3c8bd094-e165-4220-aa5d-89fc899635af)

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+
+The report can be named using the `Name of the list` field. The number of items on the report can be
+set with the field labeled: `Insert the first _ records of the list`. Next, select either a new
+blank spreadsheet, or export into an existing spreadsheet Finally, click the :guilabel:`Confirm`
```
```suggestion
:guilabel:`Blank spreadsheet`, or export into an existing spreadsheet Finally, click the :guilabel:`Confirm`
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+   :align: center
+   :alt: Set the name, number of records, and location of the export in the option menu.
+
+To export the report as a \*.xlsx file for use in an external spreadsheets program, navigate to
```
```suggestion
To export the report as a .xlsx file, for use in an external spreadsheets program, click the
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+   :alt: Set the name, number of records, and location of the export in the option menu.
+
+To export the report as a \*.xlsx file for use in an external spreadsheets program, navigate to
+:menuselection:`Settings app` and click the :icon:`fa-upload` :guilabel:`(Export All)` button. If
```
```suggestion
:guilabel:`Actions` :icon:`fa-cog` :guilabel:`(gear) icon, and select the :icon:`fa-upload` :guilabel:`Export All` option. If
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,142 @@
+
+To export the report as a \*.xlsx file for use in an external spreadsheets program, navigate to
+:menuselection:`Settings app` and click the :icon:`fa-upload` :guilabel:`(Export All)` button. If
+prompted, choose a file location and a name for the file, then click :guilabel:`Save`.
```
```suggestion
prompted, choose a file location, name the file, then click :guilabel:`Save`.
```

ksc-odoo reviewed (changes requested):
Hey @pabr-odoo -- just finished my Final Review of this PR. Great job. I have some comments/feedback that requires your attention, and there's a portion towards the end of this PR that needs some reworking. If you have any questions, don't hesitate to reach out. Once you implement the necessary adjustments, and correct that minor issue at the end, tag me again for another quick look. Thanks! 👍

pabr-odoo commented:
Hey @ksc-odoo thanks so much for the thorough review. I just implemented your suggestions. Please let me know if anything else needs changing. Thanks!

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+descending order of coverage.
+
+- *Medium* is the UTM with the widest coverage, such as social media.
+- *Source* is more narrow, and covers parameters such as the specific social media platform.
```
```suggestion
- *Source* is more narrow, and covers parameters, such as the specific social media platform.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+
+- *Medium* is the UTM with the widest coverage, such as social media.
+- *Source* is more narrow, and covers parameters such as the specific social media platform.
+- *Campaign* is the most narrow and can track specific marketing campaigns.
```
```suggestion
- *Campaign* is the most narrow, and can track specific marketing campaigns.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+       :align: center
+       :alt: Each lead is now sorted by source, followed by city or country.
+
+    This report contains all active leads, grouped first by the source of the lead, then the
```
```suggestion
    This report contains all active leads, grouped first by the source of the lead, then by the
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+    city or country each lead is from. This is useful to see the density of active opportunities
+    sorted by location.
+
+    With this data, marketing campaigns, such as conferences or billboards can be targeted to the
```
```suggestion
    With this data, marketing campaigns, such as conferences or billboards, can be targeted to the
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+==============
+
+To set the measures of the report, begin by navigating to the :icon:`oi-view-pivot`
+:guilabel:`(pivot view)`. Click the :guilabel:`Measures` button to view the available measures of
```
```suggestion
:guilabel:`(pivot view)` on the :guilabel:`Leads Analysis` dashboard.

Click the :guilabel:`Measures` button to view the available measures of
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+and verify the measures, filters, and groups are all displayed correctly in the pivot table. This
+ensures the data is ready for export.
+
+To quickly export the list as a .xlsx file, navigate to the :icon:`oi-view-list` :guilabel:`(list
```
```suggestion
To quickly export the data in a list, as a .xlsx file, navigate to the :icon:`oi-view-list` :guilabel:`(list
```

ksc-odoo replied:
kept going back-and-forth between "an .xlsx file" and "a .xlsx file" -- it all depends on if the reader reads it as "dot-xlsx" or just "xlsx" -- totally your call. Just figured it was worth mentioning...

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+right of :guilabel:`Lead Analysis` in the top-left of the page, and click :icon:`fa-upload`
+:guilabel:`Export All`. The report downloads automatically as a .xlsx file.
+
+For more export options, the report can be exported in the Odoo *Documents* app. From the
```
```suggestion
For more export options, the report can be exported to the Odoo *Documents* app. From the
```
is this correct? the original wording implies (at least to me) that I'd have to navigate/open the Documents app to complete this...

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+:guilabel:`Export All`. The report downloads automatically as a .xlsx file.
+
+For more export options, the report can be exported in the Odoo *Documents* app. From the
+:icon:`oi-view-list` :guilabel:`(list view)`, begin by clicking the :guilabel:`Actions`
```
```suggestion
:icon:`oi-view-list` :guilabel:`(list view)` of :guilabel:`Leads Analysis` page, begin by clicking the :guilabel:`Actions`
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+For more export options, the report can be exported in the Odoo *Documents* app. From the
+:icon:`oi-view-list` :guilabel:`(list view)`, begin by clicking the :guilabel:`Actions`
+:icon:`fa-cog` :guilabel:`(gear)` icon again. Now, navigate to :icon:`fa-table`
+:guilabel:`Spreadsheet` and click :icon:`oi-view-list` :guilabel:`Insert list in spreadsheet`. A
+pop-up window titled :guilabel:`Select a spreadsheet to insert your list.` appears.
```
```suggestion
:guilabel:`Spreadsheet`, and click :icon:`oi-view-list` :guilabel:`Insert list in spreadsheet`. A
pop-up window titled, :guilabel:`Select a spreadsheet to insert your list.` appears.
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+:guilabel:`Spreadsheet` and click :icon:`oi-view-list` :guilabel:`Insert list in spreadsheet`. A
+pop-up window titled :guilabel:`Select a spreadsheet to insert your list.` appears.
+
+The report can be named using the `Name of the list` field. The number of items on the report can be
```
```suggestion
The report can be renamed using the `Name of the list` field, if desired. The number of items on the report can be
```

ksc-odoo commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,157 @@
+
+The report can be named using the `Name of the list` field. The number of items on the report can be
+set with the field labeled: `Insert the first _ records of the list`. Next, select either a new
+:guilabel:`Blank spreadsheet`, or export into an existing spreadsheet Finally, click the
```
```suggestion
:guilabel:`Blank spreadsheet`, or export into an existing spreadsheet. Finally, click the
```

ksc-odoo reviewed (approved):
Hey @pabr-odoo -- great work on these revisions. only had a handful of other comments that require your attention, but none of them are difficult to manage. So, I am approving now 👍 once you implement the necessary changes, feel free to tag this for Tech Review. Thanks!

pabr-odoo commented:
Hey @samueljlieber tagging you for tech review on this when you get the chance. Thanks!

StraubCreative commented on content/applications/sales/crm/track_leads/marketing_attribution.rst:
```diff
@@ -0,0 +1,159 @@
+The UTM parameters used in a marketing attribution report are *Medium*, *Source*, and *Campaign*, in
+descending order of coverage.
+
+- *Medium* is the UTM with the widest coverage, such as social media.
+- *Source* is more narrow, and covers parameters, such as the specific social media platform.
+- *Campaign* is the most narrow, and can track specific marketing campaigns.
```
Define these more; these parameters mean very specific things.

StraubCreative reviewed (changes requested):
Hi @pabr-odoo 
I think the content came out nicely.

Please address the singular comment below regarding UTM definitions, and address all of the line-too-long or line-break issues listed here:

```
content/applications/sales/crm/track_leads/marketing_attribution.rst:136: Line too long (108/100) (line-too-long)
content/applications/sales/crm/track_leads/marketing_attribution.rst:147: Line too long (114/100) (line-too-long)
content/applications/sales/crm/track_leads/marketing_attribution.rst:6: consider moving "generation," to line 6 (early-line-breaks)
content/applications/sales/crm/track_leads/marketing_attribution.rst:33: consider moving "or" to line 33 (early-line-breaks)
content/applications/sales/crm/track_leads/marketing_attribution.rst:79: consider moving "the" to line 79 (early-line-breaks)
content/applications/sales/crm/track_leads/marketing_attribution.rst:80: consider moving "shows" to line 80 (early-line-breaks)
```

Can tag me again when ready for another look, thanks!

pabr-odoo commented:
Hey @StraubCreative thanks for taking a look at this. I just fixed the line-breaks issues and added to the UTM parameter definitions. Please take a look when you get the chance, thanks!

StraubCreative reviewed (changes requested):
Hi again @pabr-odoo 

Few more line break issues:
```
content/applications/sales/crm/track_leads/marketing_attribution.rst:6: consider moving "generation," to line 6 (early-line-breaks)
content/applications/sales/crm/track_leads/marketing_attribution.rst:33: consider moving "or" to line 33 (early-line-breaks)
content/applications/sales/crm/track_leads/marketing_attribution.rst:82: consider moving "the" to line 82 (early-line-breaks)
content/applications/sales/crm/track_leads/marketing_attribution.rst:83: consider moving "shows" to line 83 (early-line-breaks)
```

You can use the terminal command `make review` to check for these 😉

pabr-odoo commented:
Hey @StraubCreative thanks for catching the early line breaks. Everything should be resolved now

StraubCreative commented:
Looks good, nice work @pabr-odoo 👍 
.....
@robodoo r+
